### PR TITLE
Fixes #11607 - Add hammer ping to debug

### PR DESCRIPTION
**Pull request by chris1984**
Adding hammer ping to katello-debug so it gets added to foreman-debug when run.

### CODE REVIEW

**Comment by chris1984** [comment]
Moved ticket to Packaging project so the bot is happy

File: katello/katello-debug.sh [hunk 1]
```diff
@@ -125,5 +125,9 @@ echo "db.task_status.find({state:{\$ne: \"finished\"}}).pretty().shellPrint()" >
 
 add_cmd "mongo pulp_database /tmp/pulp_running_tasks.js" "pulp-running_tasks"
 
+# Hammer Ping
+
+add_cmd "hammer ping" "hammer-ping"
```
Comment (bbuckingham) [inline katello/katello-debug.sh:128]: I would probably drop the comment (ln 128) and the extra line.

**Comment by chris1984** [comment]
@bbuckingham thanks for the review, can you check it again? I made the corrections you suggested.

**Comment by bbuckingham** [comment]
ACK

File: katello/katello-debug.sh [hunk 2]
```diff
@@ -125,5 +125,7 @@ echo "db.task_status.find({state:{\$ne: \"finished\"}}).pretty().shellPrint()" >
 
 add_cmd "mongo pulp_database /tmp/pulp_running_tasks.js" "pulp-running_tasks"
 
+add_cmd "hammer ping" "hammer-ping"
```
Comment (lzap) [inline katello/katello-debug.sh:128]: Will this work for unauthorized user (without -u -p) ?

Comment (chris1984) [reply katello/katello-debug.sh:128]: @lzap it should be ok here is the output with the commit in it:

[root@centos7-katello-3-3 foreman-debug-6Jnku]# pwd
/tmp/foreman-debug-6Jnku

[root@centos7-katello-3-3 foreman-debug-6Jnku]# cat hammer-ping 
COMMAND> hammer ping

candlepin:      
    Status:          ok
    Server Response: Duration: 32ms
candlepin_auth: 
    Status:          ok
    Server Response: Duration: 27ms
pulp:           
    Status:          ok
    Server Response: Duration: 152ms
foreman_tasks:  
    Status:          ok
    Server Response: Duration: 45ms

**Pull request merged**